src/components/Users.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { fetchUsers } from "./api";
+
+const Users = ({ user, setUser }) => {
+	const [users, setUsers] = useState([]);
+	useEffect(() => {
+		fetchUsers().then((userList) => {
```

Comment:
This should have some kind of loading pattern for users with slower connections.